pkg/sql/parser/sql.y

```diff
@@ -2282,20 +2261,16 @@ preparable_stmt:
   {
     $$.val = $1.slct()
   }
-| set_session_stmt  // EXTEND WITH HELP: SET SESSION
-| set_csetting_stmt // EXTEND WITH HELP: SET CLUSTER SETTING
+| preparable_set_stmt // help texts in sub-rule
 | show_stmt         // help texts in sub-rule
 | update_stmt       // EXTEND WITH HELP: UPDATE
 | upsert_stmt       // EXTEND WITH HELP: UPSERT
 
+// TODO(knz): it's not well justified that one could
+// EXPLAIN EXECUTE.
```

Comment:
since `prepare a as explain ...` works i think we could safely leave out `explain execute` if it is complicated.

Reply:
yeah I agree but it's a separate PR I think